### PR TITLE
Feature/world information

### PR DESCRIPTION
**Pull request by Edjhernandez**
Here is the component WorldInformation, it shows general information and you can filter by specific date...

image

image

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -4,42 +4,50 @@ import './index.css'
...
-};
+}; 
+
+
 const router = createBrowserRouter([
   {
     path: "/", 
-    element: <NavbarWrapper/>,
+    element:
+    <NavbarWrapper />,
```
Comment (danyhoshi) [inline src/main.tsx:27]: I propose removing the lines 26 and 27 from here :grin:

File: src/main.tsx [hunk 2]
```diff
@@ -4,42 +4,50 @@ import './index.css'
...
+
+//import { Outlet } from 'react-router-dom'
 import Navbar from './components/Navbar'
 import Consult from './components/Consult.tsx'
 const NavbarWrapper = () => {
   return (
   <div>
       <Navbar/>
-      <Outlet/>
+      <WorldInformation/>
```
Comment (danyhoshi) [inline src/main.tsx:17]: # Really nice this component!!! 
I recomment place `<Outlet />` here and let it make :sparkles:

File: src/main.tsx [hunk 3]
```diff
@@ -4,42 +4,50 @@ import './index.css'
...
-    element: <NavbarWrapper/>,
+    element:
+    <NavbarWrapper />,
+
+    
     children: [
-      {
-      path: "/",
-      element: (
-        <h1>Home</h1>
```
Comment (danyhoshi) [inline src/main.tsx:26]: # Great job!!! 
The information is recovered correctly and displayed very well! I suggest add your `<WorldInformation/>` component to this place.  :grinning:

**Review by danyhoshi — CHANGES_REQUESTED** [review]
# It's a very nice and functional job! Congratulations! :+1:
Only a few changes

**Comment by Edjhernandez** [comment]
Hi @danyhoshi, nice observation, I did the changes you suggested... thanks. 👍🏾

**Review by danyhoshi — APPROVED** [review]
# Fine! Let's Merge!!!

**Pull request merged**